### DISCUSSION TITLE
mm-glib dependency not found

### DISCUSSION THREAD

**Discussion by wanmukhlis** [BuddiesOfBudgie/budgie-control-center#64] [General]
## Description

I tried to run 'meson build' command but i got error missing mm-glib dependency. i tried to find the dependency using 'eopkg sr' but couldn't find the dependency

## Operating system

Solus

**Comment by fossfreedom** [comment]
@EbonJaeger please can you help the OP with whatever the package name Solus uses for this dependency?

https://github.com/BuddiesOfBudgie/budgie-control-center/blob/1d24cd2c5b3a6c7f679538a66f090d0868d8d1d2/meson.build#L232

**Comment by EbonJaeger** [comment]
That would be from `modem-manager-devel`

**Comment by carlosbrad** [comment]
You could add support again for online accounts by placing the gnome-online-accounts-gtk package as a dependency, which works in any Linux desktop environment without the need to use gnome, just put a link to it in budgie-control- center to use it for online accounts

**Comment by fossfreedom** [comment]
ok - lets assume this issue has been dealt with.  Closing
